tests/integration/_cases/react_native/xcode-upload-source-maps-release_and_dist_from_env.trycmd

```diff
@@ -19,8 +19,7 @@ Processing react-native sourcemaps for Sentry upload.
 
 Source Map Upload Report
   Scripts
-    ~/react-native-xcode-bundle.js.map
-  Minified Scripts
+    ~/react-native-xcode-bundle.js.map (no sourcemap ref)
```

Comment:
It is probably a bug that this line is under "Scripts." I already have a separate issue (#1963) to investigate